helm-charts/splunk-otel-collector/templates/daemonset.yaml

```diff
@@ -74,9 +74,24 @@ spec:
       affinity:
         {{ toYaml . | nindent 8 }}
       {{- end }}
-      {{- with .Values.tolerations }}
       tolerations:
-        {{ toYaml . | nindent 8 }}
+      {{- if and (.Values.tolerations) (ne (len .Values.tolerations) 0) }}
+        {{ toYaml .Values.tolerations | nindent 8 }}
+      {{- else }}
+        - key: node-role.kubernetes.io/master
+          effect: NoSchedule
+        - key: node-role.kubernetes.io/control-plane
+          effect: NoSchedule
+        {{- if eq .Values.distribution "openshift" }}
+        - key: node-role.kubernetes.io/infra
+          effect: NoSchedule
+          operator: Exists
```

Comment:
Small nit - can we make the tolerations consistent? The other 2 tolerations before do not have an explicit `operator` field

Reply:
Sure. Done.